File: ui/src/types/team.ts
export interface Team {
    uuid: string;
    name: string;
    description: string;
    organizationID: number;
}

export interface CreateTeamRequest {
    name: string;
    description: string;
    organizationID: number;
}

export interface UpdateTeamRequest {
    name?: string;
    description?: string;
}

File: src/domain/model/team.go
package model

type Team struct {
	BaseModel
	Name           string `gorm:"size:100;not null"`
	Description    string `gorm:"size:1000"`
	OrganizationID uint   `gorm:"not null"`
	Organization   Organization
	Users          []User `gorm:"many2many:team_users"`
}
